src/evalfunc.c

```diff
@@ -5216,6 +5211,41 @@ f_get(typval_T *argvars, typval_T *rettv)
 	copy_tv(tv, rettv);
 }
 
+/*
+ * "getcellpixels()" function
+ */
+    static void
+f_getcellpixels(typval_T *argvars UNUSED, typval_T *rettv)
+{
+    if (rettv_list_alloc(rettv) == FAIL)
+        return;
+
+#if defined(FEAT_GUI)
+    if (gui.in_use)
+    {
+        // success pixel size and no gui.
+        list_append_number(rettv->vval.v_list, (varnumber_T)gui.char_width);
+        list_append_number(rettv->vval.v_list, (varnumber_T)gui.char_height);
+    }
+    else
+    {
+#endif
+        struct cellsize cs;
+        mch_calc_cell_size(&cs);
```

Comment:
I think it might be easier if you just do something like this, and don't define this for Win32:

```c
#if defined(UNIX)
    mch_calc_cell_size(&cs);
#else
    cs_out->cs_xpixel = -1;
    cs_out->cs_ypixel = -1;
#endif
```

The reason is that outside of Unix and Win32 we have platform definitions for things like VMS / Amiga. They aren't used that often or checked in CI, but unless you define this function for all platforms, this code won't build on all of them. E.g. if you look at `mch_get_shellsize` for example, it's defined for Amiga, VMS, etc. I'm not sure if you want to define all of them just to have them set xpixel to -1.

Reply:
Fixed in 44b5e6cdbc53c85377a7f8e4b90b8767551df217